doc/release-notes/release-3.0.sgml

```diff
@@ -19,7 +19,7 @@ This new release is available for download from <url url="http://www.squid-cache
 
 A large number of the show-stopper bugs have been fixed along with general improvements to the ICAP support and additional Languages.
 
-We welcome feedback and bug reports. If you find a bug, please see <url url="http://wiki.squid-cache.org/SquidFaq/BugReporting"> for how to submit a 
+We welcome feedback and bug reports. If you find a bug, please see <url url="https://wiki.squid-cache.org/SquidFaq/BugReporting"> for how to submit a 
```

Comment:
CI errors is one more reason to leave these old files alone:

```
  Error:  git diff detected bad whitespace changes.
```

This PR does not actually _add_ a trailing whitespace -- this old file is already full of those problems, and this specific line already has a trailing whitespace, but "git diff --check" is not smart enough to know that.

Reply:
Done in [9e227de](https://github.com/squid-cache/squid/pull/1448/commits/9e227deea244a6b3e0a873a8b8f1e741c2f6fdfe).